OrbitCore/CaptureData.h

```diff
@@ -148,6 +153,9 @@ class CaptureData {
   std::string process_name_;
   std::shared_ptr<Process> process_;
   absl::flat_hash_map<uint64_t, orbit_client_protos::FunctionInfo> selected_functions_;
+
+  absl::flat_hash_set<orbit_grpc_protos::TracepointInfo, HashTracepointInfo, EqualTracepointInfo>
+      selected_tracepoints_;
```

Comment:
Can you add something like
`using TracepointInfoSet = absl::flat_hash_set<orbit_grpc_protos::TracepointInfo, HashTracepointInfo, EqualTracepointInfo>;`
to TracepointCustom.h?

Maybe it makes sense to make the HashTracepointInfo, EqualTracepointInfo internal again and just expose the typedef (if that is all you need from the outside).

Reply:
I placed the implementation for the HashTracepointInfo and EqualTracepointInfo in a internal namespace and created a typedef in TracepointCustom.h for "absl::flat_hash_set<orbit_grpc_protos::TracepointInfo, HashTracepointInfo, EqualTracepointInfo>". Thanks so much for the advice!